src/Switch/Switch.tsx

```diff
@@ -64,32 +58,22 @@ export default class Switch extends React.PureComponent {
       disabled,
     } = this.props;
 
-    const {hasOutline} = this.state;
-
     return (
       <button
         aria-checked={checked}
         aria-disabled={disabled}
         aria-labelledby={ariaLabelledby}
         aria-label={ariaLabel}
-        className={classnames(cssClass.CONTAINER, className)}
-        onMouseDown={() => this._onMouseDown()}
-        onMouseUp={() => this._onMouseUp()}
-        onFocus={this._onFocus}
-        onBlur={this._onBlur}
+        className={classnames(cssClass.CONTAINER, className, "flexbox", cssClass.BG,
+          checked && cssClass.CHECKED,
+          disabled && cssClass.DISABLED,
+        )}
         role="checkbox"
```

Comment:
nit: we can switch to using the `switch` role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Switch_role